app/src/pages/inside/launchesPage/launchesPage.jsx

```diff
@@ -214,7 +214,20 @@ export class LaunchesPage extends Component {
     deleteItemsAction: () => {},
   };
 
+  componentDidUpdate() {
+    clearInterval(this.intervalId);
+
+    this.arrIdLaunchInProgress = this.props.launches
```

Comment:
I think it's better to place this in state via `computeDerivedStateFromProps` (only if launches list has changed)